diff --git a/node_modules/react-scripts/node_modules/fsevents/lib/binding/Release/node-v57-darwin-x64/fse.node b/node_modules/react-scripts/node_modules/fsevents/lib/binding/Release/node-v57-darwin-x64/fse.node
index df54326..8c78948 100755
Binary files a/node_modules/react-scripts/node_modules/fsevents/lib/binding/Release/node-v57-darwin-x64/fse.node and b/node_modules/react-scripts/node_modules/fsevents/lib/binding/Release/node-v57-darwin-x64/fse.node differ
diff --git a/node_modules/react-scripts/scripts/build.js b/node_modules/react-scripts/scripts/build.js
index a83d287..e38bee3 100644
--- a/node_modules/react-scripts/scripts/build.js
+++ b/node_modules/react-scripts/scripts/build.js
@@ -112,15 +112,6 @@ function build(previousFileSizes) {
       if (messages.errors.length) {
         return reject(new Error(messages.errors.join('\n\n')));
       }
-      if (process.env.CI && messages.warnings.length) {
-        console.log(
-          chalk.yellow(
-            '\nTreating warnings as errors because process.env.CI = true.\n' +
-              'Most CI servers set it automatically.\n'
-          )
-        );
-        return reject(new Error(messages.warnings.join('\n\n')));
-      }
       return resolve({
         stats,
         previousFileSizes,
